api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/NewClaimDTOTestDataFactory.java

```diff
@@ -51,12 +46,16 @@ public static NewClaimDTO aValidClaimDTOWithNoNullFields() {
                 .build();
     }
 
-    public static NewClaimDTO aValidClaimDTOWithEligibilityOverrideOutcome(LocalDate expectedDeliveryDate,
-                                                                           List<LocalDate> childrenDob,
-                                                                           EligibilityOutcome eligibilityOutcome) {
+    public static NewClaimDTO aValidClaimDTOWithEligibilityOverride(LocalDate expectedDeliveryDate,
```

Comment:
This should be `aValidClaimDTOWithEligibilityOutcome` as that is all you are passing in as a parameter.